ui/src/layouts/unlock.rs

```diff
@@ -234,26 +240,56 @@ pub fn UnlockLayout(cx: Scope, page: UseState<AuthPages>, pin: UseRef<String>) -
                         span {
                             get_local_text("unlock.notice")
                         }
-                    }
+                    },
                     Button {
-                            text: match account_exists.current().unwrap_or(true) {
-                                true => get_local_text("unlock.unlock-account"),
-                                false => get_local_text("unlock.create-account"),
-                            },
-                            aria_label: "create-account-button".into(),
-                            appearance: kit::elements::Appearance::Primary,
-                            icon: Icon::Check,
-                            disabled: validation_failure.current().is_some() || *cmd_in_progress.current(),
-                            onpress: move |_| {
-                                if let Some(validation_error) = validation_failure.get() {
-                                    shown_error.set(validation_error.translation());
-                                } else if let Some(e) = error.get() {
-                                    shown_error.set(e.translation());
-                                } else {
-                                    page.set(AuthPages::CreateAccount);
+                        text: match account_exists.current().unwrap_or(true) {
+                            true => get_local_text("unlock.unlock-account"),
+                            false => get_local_text("unlock.create-account"),
+                        },
+                        aria_label: "create-account-button".into(),
+                        appearance: kit::elements::Appearance::Primary,
+                        icon: Icon::Check,
+                        disabled: validation_failure.current().is_some() || *cmd_in_progress.current(),
+                        onpress: move |_| {
+                            if let Some(validation_error) = validation_failure.get() {
+                                shown_error.set(validation_error.translation());
+                            } else if let Some(e) = error.get() {
+                                shown_error.set(e.translation());
+                            } else {
+                                page.set(AuthPages::CreateAccount);
+                            }
+                        }
+                    },
+                    ContextMenu {
+                        key: "{key}-menu",
+                        id: "unlock-context-menu".into(),
+                        items: cx.render(rsx!(
+                            ContextItem {
+                                icon: Icon::Trash,
+                                danger: true,
+                                aria_label: "account-reset".into(),
+                                text: get_local_text("uplink.reset-account"),
+                                onpress: |_| {
+                                    let _ = fs::remove_dir_all(&STATIC_ARGS.dot_uplink);
```

Comment:
after removing the directory, you may want to sync the file system. You may also want to include a short delay of a few ms.